src/stores/businessStore.ts

```diff
@@ -142,13 +142,23 @@ export const useBusinessStore = defineStore('business', {
     },
 
     /** Is True if entity is a Benefit Company. */
-    isBComp (state: BusinessStateIF): boolean {
-      return (state.businessInfo.legalType === CorpTypeCd.BENEFIT_COMPANY)
+    isBComp (): boolean {
+      return (this.getLegalType === CorpTypeCd.BENEFIT_COMPANY)
```

Comment:
... And I used the getter for legal type instead of extracting it from the state.